### ISSUE TITLE
include(Sass.php): failed to open stream: No such file or directory

### ISSUE THREAD

**Issue by RdeWilde** [absalomedia/sassphp#11]
I get this error when I try to use the extension:
`include(Sass.php): failed to open stream: No such file or directory`

How can I check if the extension was installed succesfully? Should it show up in `phpinfo()` or `php -m`?

**Comment by rodrigoaguilera** [comment]
There is a `extension_loaded` function
https://www.php.net/manual/en/function.extension-loaded.php
So you can do
```
      if (!extension_loaded('sass')) {
```

**Comment by absalomedia** [comment]
It should show up in PHP Info as Sass if properly installed

**Comment by RdeWilde** [comment]
Seems like the wrong plugin version was loaded (we are running php 7.2, but the plugin wasnt).

Now I get an empty reponse. Any clue how I can find the error?

```
<?php
new Sass();
phpinfo(); // no output
?>
```

**Comment by RdeWilde** [comment]
This seems to be the error, at least that is what I found in the apache logs

```
/usr/sbin/apache2: symbol lookup error: /usr/lib/php/20170718/sass.so: undefined symbol: sass_object_fetch_object
```

I found this commit https://github.com/pilif/sassphp/commit/fb496e7d7bc8b3a5adc7d2455afb09bcf14b32df
and this one https://github.com/sensational/sassphp/commit/fcee2992570c6cf38e9bd46725028eb21df07c8f
That seem to be related

**Comment by RdeWilde** [comment]
Seems like https://github.com/absalomedia/sassphp/blob/580c176c6cf1e458ebf73b02f7ef3145a237fc20/src/sass.c#L59 should be changed to `sass_fetch_object`

or the other way around, change 
https://github.com/absalomedia/sassphp/blob/580c176c6cf1e458ebf73b02f7ef3145a237fc20/src/sass.c#L50 to `sass_object_fetch_object`

@absalomedia Could you please re-open the issue?

**Comment by absalomedia** [comment]
Keeping this open as current codebase is midway through having functions & importers included. Will close once I've got success across Travis & Appveyor

**Comment by absalomedia** [comment]
Fixed by 0.7 release
